items.yaml

```
@@ -44,7 +83,57 @@ Knight Boots:
   armor protection: .24
   agility: .2
   description: "The generic boots of any good knight."
+Orc Iron Legginsg:
+  type: "Armor Piece: Leggings"
+  armor protection: .28
+  agility: .15
+  description: "The generics boots of Goblins and Orcs. These leggings continues more longer on your legs since orcs and goblins don't wear boots.'"
+Orc Shredded Chestplate:
+  type: "Armor Piece: Chestplate"
+  armor protection: .32
+  agility: .15
+  description: "The generic boots of Goblins and Orcs."
```

```suggestion
  description: "The generic chestplate of Goblins and Orcs."
```
I'll come up with better descriptions later.